### ISSUE TITLE
genome data on sherlock

### ISSUE THREAD

**Issue by PocketChangeJ** [ENCODE-DCC/atac-seq-pipeline#12]
Hi, 
Thanks again for making your pipeline available.
I'm trying to run the wdl pipeline on sherlock, using 'sherlock.tsv' file for genome, and using shared genome data. Seems the ".tar" files for bowtie2 and bwa indexes can't be found?
Thanks!

**Comment by leepc12** [comment]
All genome data files have been recovered. Please try again.

**Comment by PocketChangeJ** [comment]
Hi Jin,
Thanks! Seems the bowtie2 step completes fine now.
I do have the pipeline crash on the next step though - it gets to *.trim.merged.filt.bam, but seems it can't find picard. I checked to be sure and I can't see it installed in the atac conda environment. (Haven't tried with my local picard installation yet).
Thanks again!

**Comment by leepc12** [comment]
How did you install dependencies? Did you follow the instruction here https://encode-dcc.github.io/wdl-pipelines/install.html#slurm?

**Comment by PocketChangeJ** [comment]
Yes. And now that you've asked I realize it should be there.. 
Let me try reinstall dependencies and try again, perhaps something went wrong there. Thanks.

**Comment by PocketChangeJ** [comment]
Hi Jin,
I reinstalled dependencies again, but the pipeline continues to crash at this step - it's still unable to find picard. I can see the jar file was indeed downloaded at the end of installation process, and I can find it in ./miniconda3/envs/encode-atac-seq-pipeline/lib/python2.7/site-packages. Conda doesn't see it as an installed package though, and I think its not in my PATH either, so perhaps that's the issue? Thanks again for the help.

**Comment by PocketChangeJ** [comment]
Hi,
Thanks, no issues with picard anymore!
I did have other things that popped out down the line, and I thought I'll just continue here.. 
First, I had to decrease memory for some of the tasks - sherlock was complaining and wouldn't submit jobs. I'm not sure if there is a way to specify that besides directly messing with the .wdl file? 
Second - it seems the pipeline can't find my idr..
It is installed and can be called from my 'encode-atac-seq-pipeline' env, no idea what's the problem.
I'm attaching stderr and stdout for this step - please let me know if there is anything else I can check.
Thank you!
[stderr.txt](https://github.com/ENCODE-DCC/atac-seq-pipeline/files/2141761/stderr.txt)
[stdout.txt](https://github.com/ENCODE-DCC/atac-seq-pipeline/files/2141762/stdout.txt)

**Comment by leepc12** [comment]
About IDR, did you run `$ source activate encode-atac-seq-pipeline` before running a pipeline?

You can change resource settings in an input JSON file. Please check section 10 of [this](https://encode-dcc.github.io/wdl-pipelines/input_json_atac.html).

**Comment by PocketChangeJ** [comment]
Hm, should have - but let me try again cause now I'm not sure .. 
And thanks for the link - seems I missed this.

**Comment by PocketChangeJ** [comment]
That was a spot on - everything run great this time.
Thanks for the help, and thanks again for doing this!
